client/client_test.go

```diff
@@ -593,4 +637,3 @@ func cleanupClients(t *testing.T, clients []*client.Client) {
 		assert.Nil(t, err)
 	}
 }
-
```

Comment:
POSIX standard that there should be a newline character at the end of every line.